pillar.example

```diff
@@ -1,10 +1,18 @@
 chrony:
   ntpservers:
-    - 'ntp0.nl.net'
-    - 'ntp1.nl.net'
-    - 'ntp2.nl.net'
-    - 'ntp.xs4all.nl'
-    - '0.nl.pool.ntp.org'
-    - '1.nl.pool.ntp.org'
-    - '2.nl.pool.ntp.org'
-    - '3.nl.pool.ntp.org'
+    - '0.debian.pool.ntp.org'
+    - '1.centos.pool.ntp.org'
+    - '2.arch.pool.ntp.org'
+    - '3.gentoo.pool.ntp.org'
+  keyfile: /etc/chrony.keys
```

Comment:
```diff
-  keyfile: /etc/chrony.keys
```

* This is needed for the test to pass, since `keyfile` is tested dependent on the `os`.

Reply:
I removed this from the pillar.example.